insights/specs/datasources/httpd.py

```diff
@@ -107,7 +107,7 @@ def get_httpd_configuration_files(httpd_root):
                     section_number = section_number + 1
                 if line.startswith("ServerRoot "):
```

Comment:
Same case here, what about change this `if` to `elif`?

Reply:
@JoySnow - updated, please check